src/main/scala/org/neo4j/spark/Neo4jOptions.scala

```diff
@@ -242,6 +263,7 @@ object Neo4jOptions {
   // session options
   val DATABASE = "database"
   val ACCESS_MODE = "access.mode"
+  val WRITE_MODE = "write.mode"
```

Comment:
solo per essere aderente alla terminologia di Spark
```suggestion
  val SAVE_MODE = "save.mode"
```